base/ltoutput.dtx

```diff
@@ -1089,7 +1089,7 @@
 %    moved this code to the newly introduced output routine sockets.
 %  \changes{v1.4q}{2025/03/20}{Remove skip added in 2.4a (footmisc/19)}
 %    \begin{macrocode}
-%  \par
```

Comment:
This needs to have a clear explanation why it is necessary even if we are already in vmode because that is not at all obvious with the normal meaning of \par.

Reply:
Also needs changes and v/date change imo

Reply:
Agreed that it needs explanation.

Reply:
done.